lib/widgets/inbox.dart

```diff
@@ -397,10 +411,14 @@ class _StreamHeaderItem extends _HeaderItem {
   @override get unreadCountBadgeBackgroundColor =>
     subscription.colorSwatch().unreadCountBadgeBackground;
 
-  @override get onCollapseButtonTap => () {
+  @override get onCollapseButtonTap => () async {
     if (collapsed) {
       pageState.uncollapseStream(subscription.streamId);
     } else {
+      await Scrollable.ensureVisible(
```

Comment:
nit in commit message:

    Fixes: zulip#391.

See our style guide on formatting this line: https://github.com/zulip/zulip-mobile/blob/main/docs/style.md#fixes-format